@navikt/core/react/src/util/virtualfocus/virtualfocus.stories.tsx

```diff
@@ -0,0 +1,381 @@
+import { Meta } from "@storybook/react";
+import React, {
+  Dispatch,
+  ReactNode,
+  SetStateAction,
+  forwardRef,
+  useRef,
+  useState,
+} from "react";
+import {
+  CloudIcon,
+  HangerIcon,
+  HourglassIcon,
+  HouseIcon,
+  PuzzlePieceIcon,
+  RulerIcon,
+} from "@navikt/aksel-icons";
+import { Button } from "../../button";
+import { Search } from "../../form/search";
+import { VStack } from "../../layout/stack";
+import { Popover } from "../../popover";
+import { useVirtualFocusDescendant } from "./Context";
+import { VirtualFocus } from "./VirtualFocus";
+
+export default {
+  title: "Utilities/VirtualFocus",
+  component: VirtualFocus,
+  decorators: [
+    (Story) => (
+      <VStack gap="10">
+        <Button>above</Button>
+        <Story />
+        <Button>below</Button>
+      </VStack>
+    ),
+  ],
+} as Meta;
+
+const MyAnchor = forwardRef<
+  HTMLInputElement,
+  {
+    children: React.ReactNode;
+    value: string;
+    setValue: Dispatch<SetStateAction<string>>;
+  }
+>(({ children, value, setValue }, ref) => {
+  const { descendants } = useVirtualFocusDescendant();
+  const searchRef = useRef<HTMLInputElement>(null);
+
+  const to_focus = descendants.item(0);
+
+  return (
+    <div style={{ position: "relative" }}>
+      <VirtualFocus.Anchor
+        role="combobox"
+        ref={ref}
+        onSelect={() => {
+          console.log(`onSelect(): ${value}`);
+        }}
+        onActive={() => {
+          console.log(`onActive(): Anchor`);
+        }}
+      >
+        <Search
+          label="Søk på nav sine sider"
+          autoComplete="off"
+          value={value}
+          onChange={(current_input) => {
+            setValue(current_input);
+            to_focus?.handleOnActive();
+          }}
+          ref={searchRef}
+        />
+      </VirtualFocus.Anchor>
+      {children}
+    </div>
+  );
+});
+const MyItem = ({
+  children,
+  icon,
+  setValue,
+}: {
+  children: string;
+  icon: ReactNode;
+  setValue: Dispatch<SetStateAction<string>>;
+}) => {
+  return (
+    <VirtualFocus.Item
+      onActive={() => {
+        setValue(children);
+        console.log(`onActive(): ${children}`);
+      }}
+      onSelect={() => {
+        console.log(`onSelect(): ${children}`);
+      }}
+      role="button"
+    >
+      <span>
+        {icon}
+        <span>{children}</span>
+      </span>
+    </VirtualFocus.Item>
+  );
+};
+
+const StoryStyle = () => (
+  <style>{`
+[data-aksel-virtualfocus="true"] {
+  outline: solid 3px var(--a-border-focus);
+  border-radius: var(--a-border-radius-medium);
+}
+`}</style>
+);
+
+export const Default = () => {
+  const [value, setValue] = useState("");
+
+  return (
+    <>
+      <StoryStyle />
+      <VirtualFocus>
+        <MyAnchor setValue={setValue} value={value}>
+          <VirtualFocus.Content>
+            <VStack>
+              <MyItem setValue={setValue} icon={<CloudIcon aria-hidden />}>
+                {"cloud "}
```

Comment:
Why is there a whitespace after each item?

Reply:
ah, this was intentional, only to make the demo for the autocomple / dropdown example to feel a bit more convenient, saving a keystroke in case you wanted to add another search term. But it doesn't really make sense regardless, since there's no "real usecase" behind this example search box that necessitates adding several terms.